src/ui/chat-window.js

```diff
@@ -89,15 +91,15 @@ export class ChatScene extends Phaser.Scene {
         this.show(false);
     }
 
-    // Takes in text and refreshes the display
+    // Takes in text, refreshes display, and shows notification button
     writeText(text, format = FONTS.ITEM_HEADER) {
-        this.show();
-        this.chatWindow.visible = true;
         this.writeStrings(
             { x: 0, text, format },
         );
         this.scrollWindow.refresh();
         this.scrollWindow.scrollToBottom();
+        this.show(false);
```

Comment:
Small thing here - if it is currently showing then it shouldn't close. Only keep closed if it is already closed

Reply:
Alright check again